### ISSUE TITLE
support ipv6 addresses parsing

### ISSUE THREAD

**Issue by hellt** [karimra/gnmic#242]
```
gnmic -a 2406:dead:beaf:2::2:57400 -u colin -p secure --insecure get --path "/state/router[router-name=Base]/origin-validation"

Error: error parsing address '2406:c800:e000:2::2:57400': address 2406:c800:e000:2::2:57400: too many colons in address
```

**Comment by karimra** [comment]
It should be supported with this notation: `gnmic -a [2406:c800:e000:2::2]:57400`
But what doesn't seem to work is adding the default port to an IPv6 address that doesn't include a port.

Let's either rename this or open a new issue.

**Comment by karimra** [comment]
should be solved by #244
